### ISSUE TITLE
tests/beakerlib/Tuned-takes-too-long-to-reload-start-when-ulimit/runtest.sh should be more clear

### ISSUE THREAD

**Issue by RHEmployee** [redhat-performance/tuned#544]
Test checks if there is delay after tuned start.

It exploits small time window among beakerlib commands which is not clear at first look.

Maybe just add `sleep .5` and more comments will help.
Or maybe use `timeout` command in combination with exploit of "race condition".

**Comment by yarda** [comment]
Yep, I think you should use the timeout command or similar mechanism. Now you are only checking that the 'sysctl' log message was removed, which IMHO was.

**Comment by RHEmployee** [comment]
Bug has been revealed in test here:
https://github.com/redhat-performance/tuned/pull/421#discussion_r1283258773

**Comment by RHEmployee** [comment]
Proposed fix:
https://github.com/redhat-performance/tuned/pull/545
